packages/vsx-registry/src/browser/vsx-extension.tsx

```diff
@@ -398,7 +403,9 @@ export abstract class AbstractVSXExtensionComponent extends React.Component<Abst
             return <div><button className="theia-button action" onClick={this.uninstall}>Uninstall</button>
                 <div className="codicon codicon-settings-gear action" onClick={this.manage}></div></div>;
         }
-        return <button className="theia-button prominent action" onClick={this.install}>Install</button>;
```

Comment:
@msujew your previous comment (https://github.com/eclipse-theia/theia/pull/10424/files#r751129871) reminds me, shouldn't we localize the buttons?

Reply:
We should, yeah. `nls.localizeByDefault` should do the trick for both of them 👍